addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -86,8 +90,15 @@ var SlideDialog = Widget.extend({
             this.reset_file();
             return;
         }
-        if (file.size / 1024 / 1024 > 15) {
-            this.display_alert(_t("File is too big. File size cannot exceed 15MB"));
+
+        // Wait for max_payload to be available
+        while (max_payload === false);
```

Comment:
I think this should not be this way, but the other option is to make `slide_upload()` fully asynchronous, and I'm not sure on how to do that.